src/main/java/com/autotune/database/dao/ExperimentDAO.java

```diff
@@ -94,4 +94,6 @@ public interface ExperimentDAO {
     // Load metadata
     List<KruizeDSMetadataEntry> loadMetadata() throws Exception;
```

Comment:
currently there no usages of `loadMetadata` function as it fetches all the records from `KruizeDSMetadataEntry` table (in case when there are multiple datasources will retrieve all the container data not limiting to a specific cluster), hence will remove the unused functionality.